index.html

```diff
@@ -1,26 +1,203 @@
+          </div>
+          <div class="paragraph-mobile">
+            <p>An annual gathering of all the best gamers every years, meet, challenge
+              learn from the leading Gamers and added tips how to make money by being a gamer. 
+            </p>
+          </div>
+          <div>
+            <h1 class="thirdheader">2022.10.14(THUR) ~ 16(FRI)</h1>
+            <p class="secondparagraph">
+              The next gamerzone event that will be held at Mainz will feature many 
+              activities at the Mainz center  as well as meeting renowned gamers live.
+            </p>
+          </div>
+        </section>
+        <section class="program">
+          <div>
+            <h1 class="programmain">Main Program</h1>
+            <hr class="new4">
+          </div>
+          <div class="programactivities">
+            <div class="programflex">
+              <i class="fa fa-gamepad fa-3x" aria-hidden="true"></i>
+              <div class="programh2"><h3>Games</h3></div>
+              <p>In our program there will be much food to eat and share with friends
+                In our program there will 
+              </p>
+            </div>
+            <div class="programflex">
+              <i class="fa fa-meetup fa-3x" aria-hidden="true"></i>
+              <div class="programh2"><h3>Online gaming</h3></div>
+              <p>In our program there will be much food to eat and share with friends
+                In our program there will 
+              </p>
+            </div>
+            <div class="programflex">
+              <i class="fa fa-grav fa-3x" aria-hidden="true"></i>
+               <div class="programh2"><h3>Virtual Reality</h3></div>
+              <p>In our program there will be much food to eat and share with friends
+                In our program there will be 
+              </p>
+            </div>
+            <div class="programflex">
+              <i class="fa fa-imdb fa-3x" aria-hidden="true"></i>
+              <div class="programh2"><h3>Movies</h3></div>
+              <p>In our program there will be much food to eat and share with friends
+                In our program there will be 
+              </p>
+            </div>
+            <div class="programflex">
+              <i class="fa fa-pied-piper-alt fa-3x" aria-hidden="true"></i>
+              <div class="programh2"><h3>Monster</h3></div>
+              <p>In our program there will be much food to eat and share with friends
+                In our program there will be much 
+            </div>
+            <div class="programflex">
+              <i class="fa fa-empire fa-3x" aria-hidden="true"></i>
+              <div class="programh2"><h3>Empire</h3></div>
+              <p>In our program there will be much food to eat and share with friends
+                In our program there will be much 
+            </div>
+          </div>
+          <button class="programbtn">JOIN GAMERZONE EVENT 2022 </button>
+        </section>
```

Comment:
- [ ] `requred`
- [ ] Kindly make the border boxes like the original design
- [ ] also please change the background, to the original on so as to follow the original design.
- [ ] Please adjust the font of  `main program`
- [ ] please see the screenshots below

- [ ] your design 
![image](https://user-images.githubusercontent.com/66230306/210707998-11ac62fc-2e2e-4902-8706-7e11fb2567cb.png)

- [ ] Behance design
![image](https://user-images.githubusercontent.com/66230306/210708981-e563bbd4-6aa2-4a8e-8455-f285b6d52b0b.png)